cement/ext/ext_smtp.py

```diff
@@ -5,10 +5,14 @@
 import smtplib
 from email.header import Header
 from email.mime.multipart import MIMEMultipart
+from email.mime.base import MIMEBase
 from email.mime.text import MIMEText
+from email import encoders
 from ..core import mail
+from ..utils import fs
 from ..utils.misc import minimal_logger, is_true
 
+
 LOG = minimal_logger(__name__)
```

Comment:
Hi @derks 

why do you prefer to create a seperate LOG in each library than using the app.log general instance?

I prefer using `self.app.log.debug()` - am I wrong?

Reply:
@TomFreudenberg extensions do not use the app logger by design, I can't recall exactly but was probably partly to clearly separate framework from application as well as being able to log before app is fully setup. By request others have asked to further remove framework logging which is why it will be disabled by default in 3.2, and why the project template disables it by default now.

I actually mention in this deprecation the idea of using app.log once it is available.

- https://docs.builtoncement.com/release-information/deprecations#id-3.0.8-2